rrmngmnt/storage.py

```diff
@@ -0,0 +1,127 @@
+import logging
+import shlex
+from rrmngmnt.service import Service
+
+logger = logging.getLogger(__name__)
+
+LV_CHANGE_CMD = 'lvchange -a %s %s/%s'
+
+
+class NFSService(Service):
+    """
+    Storage management class to maintain NFS services
+    """
+
+    def mount(self, source, target=None, opts=None):
+        """
+        Mounts source to target mount point
+
+        __author__ = "ratamir"
+        :param source: Full path to source
+        :type source: str
+        :param target: Path to target directory, if it is omitted then
+        temporary folder is created and it is the responsible of the
+        caller to remove it at the end
+        :type target
+        :param opts: List of options like: ['-t', 'nfs', '-o', 'vers=3']
+        :type opts: list
+        :return: Path to mount point if succeeded, None otherwise
+        :rtype: str
+        """
+        target = '/tmp/mnt_point' if target is None else target
```

Comment:
You may want to generate a more random mount point under /tmp in case this function is called multiple times